src/styles/main.scss

```
@@ -0,0 +1,51 @@
+.watch {
+  position: relative;
+  height: 1080px;
```

now you have vertical scroll on the page, let's add height in relative unit

```suggestion
  height: 100vh;
```